tools/apiview/parsers/CONTRIBUTING.md

```diff
@@ -31,6 +56,44 @@ Don't worry about indentation as it will be handled by the tree structure based
 
 A sample token file for .NET package `Azure.Template` is present [here](./apiview-treestyle-parser-schema/sample/Azure.Template_token.json) and corresponding rendered text representation is available [here](./apiview-treestyle-parser-schema/sample/Azure.Template_review_content.txt).
 
+See following sample API review lines and corresponding `ReviewLine` token.
+
+`namespace Azure.Data.Tables {`
+<details>
+<summary>Click here to see json</summary>
```

Comment:
because the json content is not long, will it make more sense to always have it in display instead of asking people to "See more"?

Reply:
I am planning to add more such examples. Let me change it to always visible and check.

Reply:
Done.